frontend/src/components/search/SearchCallToAction.tsx

```diff
@@ -0,0 +1,26 @@
+import "server-only";
```

Comment:
This is all static content so it's a Next server component

Reply:
Will this let us use i18n?